tests/api_workflow/mocked_api_workflow_client.py

```diff
@@ -181,6 +181,13 @@ def upsize_tags_by_dataset_id(self, body, dataset_id, **kwargs):
         _check_dataset_id(dataset_id)
         assert body.upsize_tag_creator == TagCreator.USER_PIP
 
+    def create_tag_by_dataset_id(self, body, dataset_id, **kwargs):
+        _check_dataset_id(dataset_id)
+        tag = TagData(id='inital_tag_id', dataset_id=dataset_id, prev_tag_id=None,
+                      bit_mask_data="0xF", name=body['name'], tot_size=10,
```

Comment:
You could also use 'bit_mask_data=body['bit_mask_data']' here just like for the name. The same goes for the `prev_tag_id`.